src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,180 @@
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 04 Verify response code when GET a deleted resource
+            # The id value should be replaced with the resource created in the Background section
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 05 Verify response when deleting a referenced school
+            Given the system has these "schools" references
+                  | schoolId | nameOfInstitution | gradeLevels                                                                      | educationOrganizationCategories                                                                                        |
+                  | 4003     | Test school       | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Tenth Grade"} ] | [ {"educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#school"} ] |
+              And the system has these "schoolYearTypes"
+                  | schoolYear | currentSchoolYear | schoolYearDescription |
+                  | 2022       | false             | School Year 2022      |
+              And the system has these "gradingPeriods"
+                  | schoolReference    | schoolYearTypeReference | gradingPeriodDescriptor                                 | gradingPeriodName              | beginDate  | endDate    | periodSequence | totalInstructionalDays |
+                  | {"schoolId": 4003} | {"schoolYear": 2022}    | uri://ed-fi.org/GradingPeriodDescriptor#First Six Weeks | 2021-2022 Fall Semester Exam 1 | 2021-08-23 | 2021-10-03 | 1              | 29                     |
+
+             When a DELETE request is made to referenced resource "/ed-fi/schools/{id}"
+             Then it should respond with 409
+              And the response body is
+                  """
+                    {
+                        "detail": "The requested action cannot be performed because this item is referenced by existing GradingPeriod item(s).",
+                        "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
+                        "title": "Dependent Item Exists",
+                        "status": 409,
+                        "correlationId": null,
+                        "validationErrors": {},
+                        "errors": []
+                    }
+                  """
+
+        @ignore
+        Scenario: 06 Verify response when deleting a referenced schoolyeartype
```

Comment:
Same as above - does not belong in this file.

Reply:
Removed